internal/faustjs.org/docs/gutenberg/tutorial/create-a-block-from-wordpress-core.mdx

```diff
@@ -93,54 +98,65 @@ This will resolve into:
       ]
     }
 ```
-Notice that with some of the attributes like `backgroundColor`, the editor used the `theme.json` palette slug name instead of the actual value.
-It also stored the custom hardcoded styles in the `style` fields as a JSON string. We will have to handle them appropriately when developing the associated block.
-Once you get familiar with how to query the necessary data from that block, let's see now how to implement the block in the Decoupled site.
 
-## 5. Create the initial block
+:::note
+Notice that with some of the attributes like `backgroundColor`, the editor used the `theme.json` palette slug name instead of the actual value, i.e. `tertiary`. It also stored the custom hardcoded styles in the `style` fields as a JSON string. We will have to handle them appropriately when developing the associated block.
+
+Once you get familiar with how to query the necessary data from that block, let's see how to implement the block in your Faust Headless site.
+:::
+
+## 4. Create the initial block and log the attributes
+
+Follow the steps below to create a `CoreCode` block that logs its attributes in the console.
+
+### 4.1 Create a `wp-blocks` folder in your Headless Site root
 
-Inside the codebase of your application, create the basic block structure that simply prints the block's attributes.
-The steps required to achieve that are the following:
+<img src="/docs/img/wp-blocks-folder.png" alt="wp-blocks folder" />
 
-### 5.1 Create the block and save it inside the `wp-blocks` folder
+### 4.2 Create the `CoreCode.js` inside the `wp-blocks` folder
 
 ```jsx title="wp-blocks/CoreCode.js"
+// import react
+import React from 'react';
+// create CoreCode component
 export default function CoreCode(props) {
   console.log(props.attributes);
-  return <div>CoreCode</div>
+  return <div>CoreCode</div>;
 }
 ```
 
-This component prints the block attributes via the props passed as a parameter.
+This component logs the block attributes to the console via the `attributes` prop passed through the `props` parameter.
 
-### 5.2 Export the block in the `index.js`
+### 4.3 Create an `index.js` file inside the `wp-blocks` folder that imports and then exports the `CoreCode` block. You will use this same file to export other blocks as well.
 
 ```jsx title="wp-blocks/index.js"
+// import CoreCode
 import CoreCode from './CoreCode';
+// export blocks
 export default {
-  CoreCode
-}
+  CoreCode,
+};
 ```
 
-By exporting the block in the `index.js` we make it available in the `WordPressBlocksProvider`.
-You can also use a different name as long as it matches either the Block `name` or the `__typename` fields.
-
-This also works:
-
-```javascript title="wp-blocks/index.js"
-import CoreCode from './CoreCode';
-export default {
-  'core/code': CoreCode
-}
-```
+:::note
+By exporting the block in `index.js` we make it available to the `WordPressBlocksProvider`.
+:::
```

Comment:
Docusaurus admonition should have a full line of space between the opening and closing tags to [prevent issues with prettier](https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier).

```suggestion
:::note

By exporting the block in `index.js` we make it available to the `WordPressBlocksProvider`.

:::
```